src/univariate/continuous/maxwellboltzmann.jl

```diff
@@ -0,0 +1,61 @@
+immutable MaxwellBoltzmann <: ContinuousUnivariateDistribution
+    a::Float64 # Scale factor
+    chisqd::Chisq
+
+    function MaxwellBoltzmann(a::Real)
+        a > zero(a) || error("MaxwellBoltzmann: a must be positive")
+        @compat new(Float64(a), Chisq(3.0))
+    end
+end
+
+@distr_support MaxwellBoltzmann 0.0 Inf
+
+#### Parameters
+
+scale(d::MaxwellBoltzmann) = d.a
+params(d::MaxwellBoltzmann) = (d.a,)
+
+#### Statistics
+
+mean(d::MaxwellBoltzmann) = 1.5957691216057308 * d.a # 2*sqrt(2/π)
+median(d::MaxwellBoltzmann) = 1.5381722544550522 * d.a # quantile(d, 0.5)
+mode(d::MaxwellBoltzmann) = sqrt2 * d.a # sqrt(2) = 1.4142135623730951
+
+var(d::MaxwellBoltzmann) = 0.4535209105296745 * d.a^2 # (3.0π - 8.0)/π
+std(d::MaxwellBoltzmann) = 0.6734396116428515 * d.a
+
+skewness(d::MaxwellBoltzmann) = 0.4856928280495921 # 2.0 * sqrt(2.0) * (16.0 - 5.0π)/(3π - 8)^(3/2)
+kurtosis(d::MaxwellBoltzmann) = 0.10816384281628826 # 4.0 * (-96.0 + 40.0π - 3π^2)/(3.0π - 8.0)^2
+
+entropy(d::MaxwellBoltzmann) = 0.9961541981062054 + log(d.a) # log(sqrt(2π)) + γ - 0.5
```

Comment:
Unfortunately, we cannot fold `log` yet so here the value is also required (if this function really has to be that fast).